static/js/services/languages.js

```diff
@@ -0,0 +1,20 @@
+var _ = require('underscore');
+
+/*
+ * Get all enabled languages
```

Comment:
that works too. 
In that case, moving these services to their own file is tech debt!